### ISSUE TITLE
transfer learning the pre-trained model

### ISSUE THREAD

**Issue by cvtower** [elcorto/imagecluster#6]
Hi @elcorto ,

This repo a nice work!
I applied this work to my dataset and find it works fine.
Then i replace the vgg 16 model with vgg19/res50...and almost all pretrained model provided by keras. My view is that transfer learning the model to private dataset(different class number) will be better, since the cluster number of practical data to be processed is unknown and could be quite different from imagenet.
Besides, the distance matrix could be alternative type according to type of task.

Have you ever did further research on this repo?
Thanks very much for your contribution!

**Comment by elcorto** [comment]
Hi, sorry for the late answer. As for transfer learning, no I did not try it, because I would need to label my own data, which is what I'd like to avoid doing in the first place (see also the last part of the README). It might be however sufficient to label a small subset of own data and re-train with that. People also suggested to automatically label own data using timestamp and GPS data of the images. I don't use this "in production" right now so I havent explored that. If people experiment in that direction, I'd like to hear how it goes.

I'm not sure I understand the part about cluster numbers. The number is by definition unknown, that is why you can play with the `sim` parameter instead.

Can you clarify the distance matrix bit? In case you mean distance metric, 'euclidean' or otherwise, then yes I played with several options in `scipy.spatial.distance.pdist` and found no significant difference (no serious study though). If you know a certain metric that works much better, I'd be glad to try it.

**Comment by elcorto** [comment]
I'm taking the liberty to close this one since it is not really an issue with the code. You may of course still add comments.

**Comment by elcorto** [comment]
Close due to inactive OP.
